students/Dkuchan/session08/frcscouter.py

```diff
@@ -0,0 +1,145 @@
+# FRC SCOUTER
+# frcscouter.py
+
+class Competition():
+    # this class holds information specific to a certain competition
+    def __init__(self, cname):
+        self.compname = cname
+        self.teamroster = []
+
```

Comment:
If you just have an init with a couple of variables, you might as well use a dictionary instead.